game/managers/animation-manager.js

```diff
@@ -89,6 +89,27 @@ class UILayer {
   }
 }
 
+class GameLayer {
+  constructor() {
+    this.drawables = [];
+  }
+
+  register(game) {
```

Comment:
Design: Game layer drawables should be independent of GameManager --> Possible to remove GameManager import